kola/tests/coretest/core.go

```diff
@@ -72,7 +72,8 @@ func init() {
 			"Useradd":          TestUseradd,
 			"MachineID":        TestMachineID,
 		},
-		Distros: []string{"cl"},
+		Distros:   []string{"cl"},
+		Offerings: []string{"basic", "pro"},
```

Comment:
I'm a bit confused here, do we really need to specify this field for each and every test? We don't do this for Architectures or Channels... Why do we need to do it for Offerings?

Reply:
I've updated `Offerings` similar to how `Architectures` and `Channels` are. I was thinking for a future case, where we have more offers, and miscalculated. Sorry for the confusion.